src/viewer.ts

```diff
@@ -9,6 +9,9 @@ import { PlayerAnimation } from "./animation.js";
 import { BackEquipment, PlayerObject } from "./model.js";
 import { NameTagObject } from "./nametag.js";
 
+//james090500
+import { loadCapeToCanvas } from "./skinview-overrides.js";
```

Comment:
This probably should be added to `skinview-utils` right?

Reply:
Moved to https://github.com/bs-community/skinview-utils/pull/15